Games/core/Game.js

```diff
@@ -796,7 +798,25 @@ module.exports = class Game {
     }
 
     createNextStateTimer(stateInfo) {
-        this.createTimer("main", stateInfo.length, () => this.gotoNextState());
+        this.createTimer("main", stateInfo.length, () => this.checkVeg());
+        this.checkAllMeetingsReady();
+    }
+
+    checkVeg() {
+        this.clearTimer("main");
+        this.vegMeeting = this.createMeeting(VegReadyMeeting, "vegKickMeeting");
+        for (let player of this.players) {
+            if (player.alive) {
+                this.vegMeeting.join(player);

```

Comment:
if a person hasn't voted, should they be let into the veg meeting?
it gets more complex if you want to dynamically add them in the veg meeting after they voted.